src/v0/destinations/braze/transform.js

```diff
@@ -223,40 +194,77 @@ function getUserAttributesObject(message, mappingJson, destination) {
       traits,
       data,
       message.properties?.mergeObjectsUpdateOperation,
-      destination.Config.enableNestedArrayOperations,
+      destination?.Config.enableNestedArrayOperations,
     );
   }
 
   return data;
 }
 
-function processIdentify(message, destination) {
+/**
+ * makes a call to braze identify endpoint to merge the alias (anonymousId) user with the
+ * identified user with external_id (userId) [Identity resolution]
+ * https://www.braze.com/docs/api/endpoints/user_data/post_user_identify/
+ *
+ * @param {*} message
+ * @param {*} destination
+ */
+async function processIdentify(message, destination) {
   // override userId with externalId in context(if present) and event is mapped to destination
   const mappedToDestination = get(message, MappedToDestinationKey);
   if (mappedToDestination) {
     adduserIdFromExternalId(message);
   }
 
-  return buildResponse(
-    message,
-    destination,
-    getIdentifyPayload(message),
-    getIdentifyEndpoint(destination.Config.endPoint),
-  );
+  const identifyPayload = getIdentifyPayload(message);
+  const identifyEndpoint = getIdentifyEndpoint(getEndpointFromConfig(destination));
+
+  const brazeIdentifyResp = await httpPOST(identifyEndpoint, identifyPayload, {
```

Comment:
Can we make use of the function `handleHttpRequest`?
As it will combine `httpPOST`, `httpGET` etc., with `processAxiosResponse`, which will make the code looks a lot cleaner

Reply:
Addressed